### PR TITLE
Use HTTPModuleCog for module manager

### PR DESCRIPTION
**Pull request by Fripe070**

### CODE REVIEW

File: breadcord/core_modules/module_manager/__init__.py [hunk 1]
```diff
@@ -31,7 +29,7 @@ async def autocomplete(self, interaction: discord.Interaction, value: str, /) ->
             )
             for module in breadcord.helpers.search_for(
                 query=value,
-                objects=list(interaction.client.modules),
+                objects=tuple(interaction.client.modules),
```
Comment (m31-galaxy) [inline breadcord/core_modules/module_manager/__init__.py:32]: Why this change, is it minorly more performant?

**Review by m31-galaxy — APPROVED** [review]

File: breadcord/core_modules/module_manager/__init__.py (hunk 1 above)
Comment (Fripe070) [reply breadcord/core_modules/module_manager/__init__.py:32]: Yes, but in reality I think one could just remove the conversaion altogether

Comment (m31-galaxy) [reply breadcord/core_modules/module_manager/__init__.py:32]: I'll take a look Soon™️
